### DISCUSSION TITLE
Search box diappears when using git-revision-date-localized

### DISCUSSION THREAD

**Discussion by ghost** [squidfunk/mkdocs-material#4274] [Q&A] [answered]
Hi there,

I'm using MkDocs Material for documentation and it has been a great experience so far, but I've just run into an issue that hopefully someone can help me with.

I would like to show the creation and latest change date on the page and I found that I should be able to do this with `git-revision-date-localized`. This works, but when I the plugin to my `mkdocs.yml` file, the search box at the top disappears. Does anyone have an idea how I can keep both?

Here's the code that I added in my `mkdocs.yml` file:
```
plugins:
  - git-revision-date-localized:
      type: date
```

Thanks a lot for the help!

**Comment by blueswen** [comment, marked as answer]
Try to add search plugin to your plugins back as follow:

```yml
plugins:
  - search
  - git-revision-date-localized:
      type: date
```

**Reply by ghost** [reply]
That worked, thanks a lot for the quick help!
